### DISCUSSION TITLE
英雄旧忆 (Chronicon) 1.52.0

### DISCUSSION THREAD

**Discussion by zvipid** [Gamuxorg/bbs#442] [投递和请求游戏]
游戏名称：英雄旧忆
游戏名称(英文)：Chronicon
下载地址 :（如果有的话）：https://www.mediafire.com/file/ptgq0o9yw6i9osh

这款游戏已经有 Mac 版本了， 希望能移植到 Linux 环境来， 谢谢！
Mac 版本：https://www.yuanyz6.cn/macapp/29030.html

**Comment by hyh19962008** [comment]
这个Steam上看已经支持Linux了呢

________________________________
From: zvipid ***@***.***>
Sent: Monday, October 21, 2024 10:35:32 AM
To: Gamuxorg/bbs ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [Gamuxorg/bbs] 英雄旧忆 (Chronicon) 1.52.0 (Discussion #442)


游戏名称：英雄旧忆
游戏名称(英文)：Chronicon
下载地址 :（如果有的话）：https://www.mediafire.com/file/ptgq0o9yw6i9osh

这款游戏已经有 Mac 版本了， 希望能移植到 Linux 环境来， 谢谢！
Mac 版本：https://www.yuanyz6.cn/macapp/29030.html

―

**Comment by trial** [comment]
有没有无需会员登陆可以直接下载的mac版
